src/services/auth.ts

```diff
@@ -15,7 +16,8 @@ function checkAuth(req: Request, res: Response, next: NextFunction) {
     if (!sqlInit.isDbInitialized()) {
         res.redirect("setup");
     } else if (!req.session.loggedIn && !isElectron && !noAuthentication) {
-        res.redirect("login");
+        const redirectToShare = options.getOption("redirectBareDomain") === "true";
```

Comment:
since getOption should return a boolean, you can drop the comparison entirely here.